pydantic/_internal/_decorators.py

```diff
@@ -59,8 +59,9 @@ def __init__(
         fields: tuple[str, ...],
         # pre=True/False in v1 should be converted to mode='before'/'after' in v2
         mode: Literal['before', 'after'],
-        check_fields: bool | None,
         each_item: bool,
+        always: bool,
+        check_fields: bool | None,
```

Comment:
While adding always, I just put the arguments in the same order here as they are in the actual decorator definition